### PR TITLE
Bug 1491070 - run tests and lint in Taskcluster

### PR DESCRIPTION
**Pull request by djmitche**
I tried using `xvfb-run` instead, but it just hung rather than running `yarn test`.

I also included `yarn test` and `yarn lint` in the same task because the long pole is the `yarn` (install) invocation.  Once that's done, the tests and lint are just a few seconds each.

### CODE REVIEW

**Comment by djmitche** [comment]
I'll address that error and request review again..

**Comment by djmitche** [comment]
Great success!

File: .taskcluster.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  - taskId: {$eval: as_slugid("tests")}
```
Comment (helfi92) [inline .taskcluster.yml]: Out of curiosity, where is `as_slugid` defined? I wasn't able to find it in https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v0.

Comment (djmitche) [reply .taskcluster.yml]: It's in https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v1 :)

Comment (helfi92) [reply .taskcluster.yml]: Ahh. Btw, https://tools.taskcluster.net/quickstart defaults to version 0. Should this be updated to version 1?

**Pull request merged**